test/unit/controllers/CustomsDeclarationsMetricsControllerSpec.scala

```diff
@@ -37,8 +36,7 @@ import util.TestData._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Try
 
-class CustomsDeclarationsMetricsControllerSpec extends UnitSpec
-  with Matchers with MockitoSugar {
+class CustomsDeclarationsMetricsControllerSpec extends UnitSpec with MockitoSugar {
```

Comment:
Duplicate `MockitoSugar` as its already coming from `UnitSpec`

Reply:
sure, will remove it